lang/cs/.nuget/NuGet.targets

```diff
@@ -128,13 +128,17 @@ under the License.
         <DownloadNuGet OutputFilename="$(NuGetExePath)" Condition=" '$(DownloadNuGetExe)' == 'true' AND !Exists('$(NuGetExePath)')" />
     </Target>
 
-    <Target Name="RestorePackages" DependsOnTargets="CheckPrerequisites">        
+    <Target Name="RestorePackages" DependsOnTargets="CheckPrerequisites">
         <Exec Command="$(RestoreCommand)"
               Condition="'$(OS)' != 'Windows_NT' And Exists('$(PackagesConfig)')" />
 
         <Exec Command="$(RestoreCommand)"
               LogStandardErrorAsError="true"
               Condition="'$(OS)' == 'Windows_NT' And Exists('$(PackagesConfig)')" />
+
+        <!-- Restore solution-level packages as well -->
+        <Exec Command="$(NuGetCommand) restore $(SolutionDir)\.nuget\packages.config -o $(PackagesDir)"
```

Comment:
All package dlls are missing from the references when open the VS solution first time before building. Do we need to add restore in each project?